unity/EditorPlugin/PluginSettings.cs

```diff
@@ -318,7 +321,17 @@ private static void RiderPreferencesItem()
       var githubRepo = "https://github.com/JetBrains/resharper-unity";
       var caption = $"<color=#0000FF>{githubRepo}</color>";
       LinkButton(caption: caption, url: githubRepo);
-
+      
+      GUILayout.FlexibleSpace();
+      GUILayout.BeginHorizontal();
+      
+      GUILayout.FlexibleSpace();
+      Assembly assembly = Assembly.GetExecutingAssembly();
+      FileVersionInfo versionInfo = FileVersionInfo.GetVersionInfo(assembly.Location);
```

Comment:
I think we should use `assembly.GetName().Version`. This is the ".net version" as it were, and we've had issues before where we've had the assembly version correct but not the file version.